### PR TITLE
William McCullen HW #01

### PR DESCRIPTION
**Pull request by wmccullen**
I didn't find this assignment to be too difficult. I wasn't able to find out how to make my image work, so I simply made a link instead. I used a guide on some of the basic syntax, which helped me a lot!

### CODE REVIEW

**Comment by wanitchayap** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |     X         |            |
| **Coding strategy**         |           |      X        |            |
| **Presentation**            |           |        X      |            |
| **Achievement, creativity** |           |      X        |            |
| **Reproducibility**         |     X      |              |            |

Remarks:
* Great job :D 
* Commit often with detail commit messages 👍 
* For the link to work properly, you need to have no space between [] and ()
```
[link](https://shellshock.io)
```
instead of 
```
[link] (https://shellshock.io)
```

File: README.md [hunk 1]
```diff
@@ -1 +1,14 @@
...
+My name is <strong>William McCullen</strong>.
+I am a 17 year old student and an incoming Senior in high school.
+Some of my favorite activities are:
+<ol>
+<li>Soccer</li>
+<li>Video Gaming</li>
+<li>Board/Card Games</li>
+</ol>
+Here is a link to a cool game that my friends and I play sometimes:
+[link](https://shellshock.io)
```
Comment (wanitchayap) [inline README.md:14]: For some reasons, this link syntax isn't rendered properly.

File: README.md [hunk 2]
```diff
@@ -1 +1,14 @@
-# hw01
\ No newline at end of file
+# hw01
+
+<h1>My Homework #1</h1>
```
Comment (wanitchayap) [inline README.md:3]: You can simply use `# My Homework #1` too!

File: README.md [hunk 3]
```diff
@@ -1 +1,14 @@
...
+
+My name is <strong>William McCullen</strong>.
```
Comment (wanitchayap) [inline README.md:5]: `**William McCullen**` will give you bold as well

File: README.md [hunk 4]
```diff
@@ -1 +1,14 @@
...
+I am a 17 year old student and an incoming Senior in high school.
+Some of my favorite activities are:
+<ol>
+<li>Soccer</li>
```
Comment (wanitchayap) [inline README.md:9]: This also can be simply done by
```
* Soccer  
* Video Gaming
* Board/Card Games
```

**Review by wanitchayap — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      X     |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |           |        X      |            |
| **Achievement, creativity** |           |      X        |            |
| **Reproducibility**         |     X      |              |            |

Remarks:
* Great job :D 
* Commit often with detail commit messages 👍 
* Markdown elements can be implemented with simpler syntax

File: README.md [hunk 5]
```diff
@@ -1 +1,14 @@
-# hw01
\ No newline at end of file
+# hw01
+
+<h1>My Homework #1</h1>
+
```
Comment (klyzhng) [inline README.md:4]: Tip: you can use more #'s to get smaller sized headers. So "# hello" would display larger than "### hello".

File: README.md (hunk 1 above)
Comment (klyzhng) [inline README.md:14]: I googled a YouTube video to find how to embed links in Markdown but I'm not sure why your code (seems to be written correctly) didn't display the link properly. You mentioned that you wanted to embed an image; I did this by following example code from https://guides.github.com/features/mastering-markdown/. Scroll down to "Examples" and click "Images". There is also more info further down the page. If it helps, you can take a look at my code (last line). I tried pasting the code here but it automatically see the image so you wouldn't be able to see the code.

File: README.md [hunk 6]
```diff
@@ -1 +1,14 @@
-# hw01
\ No newline at end of file
+# hw01
+
```
Comment (klyzhng) [inline README.md:2]: I didn't know that you could code in R with these arrows <> and slashes, so I learned something new thanks to you! And I agree with everything wanitchayap has said but refrained from repeating the comments. It was cool seeing your work and kudos to you on learning some R before college -- it's definitely fun and useful!

File: README.md (hunk 2 above)
Comment (hanjimin06) [reply README.md:3]: Second this!

**Comment by hanjimin06** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       *       |            |
| **Coding strategy**         |           |       *       |            |
| **Presentation**            |     *      |              |            |
| **Achievement, creativity** |     *      |           |            |
| **Reproducibility**         |           |      *        |            |

Remarks:

It was a very interesting introduction. it had more than just a general overview, and it was also interesting to read. I also noticed that it did include most of the elements required, which was also a plus.

For improvement:

1. I believe the style of the code could use some improvement, as it could be made easier for both the writer and the readers. But it's good that you looked up some syntax guides. 

2. More commits! I feel that habit would help later on when we're asked to do more complicated coding.

3. I also had some trouble embedding an image at first, but it turned out to be easier than I thought, so it'd be good to try it out.

**Pull request closed without merging**